src/clarity-angular/tabs/vtabs-content.ts

```diff
@@ -0,0 +1,42 @@
+import {Component} from "@angular/core";
+
+
+/**
+ * @class VTabContent
+ *
+ * @description
+ * A child component of VTabs that:
+ * 1. Provides an id property set and used by the VTabs class to map VTabsNavLink items to the corresponding
+ *    VTabsContent items
+ * 2. Provides an activated property that triggers Angular to add/remove the content to the DOM via *ngIf
+ */
+@Component({
+    selector: "clr-vtabs-content",
+    template: `
+        <section class="vtabs-content" *ngIf="activated" role="tabpanel">
```

Comment:
So this may be something I take issue with....

In the DOM example above, we are using `aria-hidden="true"` to hide `.vtabs-content`. But here we are using `*ngIf`.

First, I have an issue with presenting two different ways to do the same thing.

Second, I have questions about how using `*ngIf` in this manner affects accessibility.

Is there a specific reason we are using `*ngIf="activated"` instead of `[attr.aria-hidden]="activated"`?